paper/paper.md

```diff
@@ -58,14 +58,12 @@ pyGeo is a geometry package for three-dimensional shape manipulation tailored fo
 It provides basic geometry generation capabilities, several methods for geometry parameterization, numerous geometric constraints, and utility functions for geometry manipulation.
 The code computes derivatives for all parameterization methods and constraint functions, enabling gradient-based optimization.
 <!--MM: I am team parameTRIzation-->
+<!--AY: I have more commonly used and seen "parameterization" so I changed the remaining instances of parametrization to parameterization-->
```

Comment:
Fine with me

Reply:
Well it is _parameter_ so you know :stuck_out_tongue: